tools/nnyeah/tests/XamarinDependencyTests.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading.Tasks;
+using System.IO;
+using NUnit.Framework;
+using Mono.Cecil;
+using Microsoft.MaciOS.Nnyeah;
+using Xamarin;
+
+namespace Microsoft.MaciOS.Nnyeah.Tests {
+	[TestFixture]
+	public class XamarinDependencyTests {
+		[Test]
+		public async Task HasReferenceTest ()
+		{
+			var dir = Cache.CreateTemporaryDirectory ("BasicLibrary");
```

Comment:
This string should be unique for each test. If set to null it walks up to the async wrapper of the test and creates a "funny" name, but we want each test to be independent.